profiles/community/init.sh

```diff
@@ -0,0 +1,14 @@
+# #!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+source /src/galaxy_ng/profiles/base/init.sh
+
+cp -r /src/galaxy_ng/profiles/community/galaxy-importer /etc/galaxy-importer
+
+setup_collection_signing_service
+
+# collect rest_framewrok static files
+django-admin collectstatic --no-input --clear
```

Comment:
This needs to be added to oci-env itself. It got removed at some point and I don't know why

Reply:
It's still being used here, but I didn't want to include the UI part, since the UI configuration doesn't work with community (`API_PATH_PREFIX` is different)
https://github.com/ansible/galaxy_ng/blob/5c301f69c1075700758f5e8db783c66f20d085bb/profiles/base/init.sh#L104-L108